week-1/2-mandatory/3-function-output.js

```diff
@@ -1,21 +1,39 @@
 // Add comments to explain what this function does. You're meant to use Google!
+/*
+* getNumber() returns a random number within the range 0 < x < 10.
+*
+ * The Math.random() function returns a floating-point number in the range 0 < x 1.
+ * 
+ * Before the number is returned it is multiplied by ten thus extending the range.
+ * 
+ */
 function getNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+/*
+ * The s(w1, w2) function accepts two strings and returns a new string by concatenating
+ * the argumnents w1 and w2
+ * e.g if the function is call with the following arguments: s("Hello", " World!"), where the
+ * second argument is prepended with a space, then the function will return the following string:
+ * "Hello World!"
+ */
```

Comment:
FYI - these could potentially be arrays also :)
`[1, 2, 3].concat([4, 5, 6]) // returns [1, 2, 3, 4, 5, 6]`